internal/productcatalog/feature_connector.go

```diff
@@ -0,0 +1,151 @@
+package productcatalog
+
+import (
+	"context"
+	"slices"
+
+	"github.com/openmeterio/openmeter/internal/meter"
+	"github.com/openmeterio/openmeter/pkg/framework/entutils"
+	"github.com/openmeterio/openmeter/pkg/models"
+)
+
+type CreateFeatureInputs struct {
+	Name                string             `json:"name"`
+	Key                 string             `json:"key"`
+	Namespace           string             `json:"namespace"`
+	MeterSlug           string             `json:"meterSlug"`
+	MeterGroupByFilters *map[string]string `json:"meterGroupByFilters"`
```

Comment:
Map itself is a pointer. If you just remove the star most of the code will just work fine. I have seen that oapi codegen does the same thing, but still the pointer is unnecessary.